{
  "Header": {
    "Original": "CVE-2024-8445 (The fix for CVE-2024-2199 in 389-ds-base was insufficient to cover all ...)",
    "ID": "CVE-2024-8445",
    "Description": "(The fix for CVE-2024-2199 in 389-ds-base was insufficient to cover all ...)"
  },
  "Annotations": [
    {
      "Original": "- 398-ds-base \u003cunfixed\u003e",
      "Type": "package",
      "Package": "398-ds-base",
      "Kind": "unfixed"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=2310110",
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=2310110"
    },
    {
      "Original": "NOTE: CVE exists because of an insufficent/incomplete fix for CVE-2024-2199",
      "Type": "NOTE",
      "Description": "CVE exists because of an insufficent/incomplete fix for CVE-2024-2199"
    },
    {
      "Original": "TODO: check details",
      "Type": "TODO",
      "Description": "check details"
    }
  ]
}